### ISSUE TITLE
[Bug]: Only loads when Remindertab is open

### ISSUE THREAD

**Issue by foreversun69** [uphy/obsidian-reminder#205]
### Describe the bug

Recently, the reminder pop-ups only load when the reminder tab is actively open.

### Expected Behavior

The pop-ups should also trigger without an active reminder tab, as they used to.

### Steps to reproduce

![grafik](https://github.com/user-attachments/assets/8f1ea078-42cb-4aa1-82a2-10c3684aa930)

### Operating system

Linux

**Comment by brainwo** [comment]
Same issue with same operating system
Digging a little bit deeper, this error shows up in console log:

```
Uncaught (in promise) TypeError: view.reload is not a function
    at eval (plugin:obsidian-reminder-plugin:16775:38)
    at Array.forEach (<anonymous>)
    at ReminderListItemViewProxy.reload (plugin:obsidian-reminder-plugin:16775:15)
    at ReminderPlugin.eval (plugin:obsidian-reminder-plugin:17008:22)
    at Generator.next (<anonymous>)
    at eval (plugin:obsidian-reminder-plugin:50:61)
    at new Promise (<anonymous>)
    at __async (plugin:obsidian-reminder-plugin:34:10)
    at ReminderPlugin.periodicTask (plugin:obsidian-reminder-plugin:17007:12)
    at ReminderPlugin.startPeriodicTask (plugin:obsidian-reminder-plugin:16988:10)
 ```

**Comment by tescher** [comment]
I have this issue as well. No reminders are popping up, ever. Console log is filled with this error. Windows 11, Reminder V1..1.15, Obsidian v1.7.7

**Comment by uphy** [comment]
Could you try to use v1.1.16?

**Comment by brainwo** [comment]
It's fixed in v1.1.6, thanks for the prompt @uphy
